stable/phpmyadmin/README.md

```diff
@@ -44,27 +44,28 @@ The command removes all the Kubernetes components associated with the chart and
-| `image.repository`                   | phpMyAdmin Image name                     | `bitnami/phpmyadmin`                                     |
-| `image.tag`                          | phpMyAdmin Image tag                      | `{VERSION}`                                             |
-| `image.pullPolicy`                   | Image pull policy                        |   `IfNotPresent` |
-| `image.pullSecrets`                  | Specify image pull secrets               | `nil`                                                   |
-| `service.type`            | type of service for phpMyAdmin frontend             | `ClusterIP`                                                  |
-| `service.port`        | port to expose service                   | `80`                                                   |
-| `db.port`            | database port to use to connect                  | `3306`                                     |
-| `db.chartName`                | Database suffix if included in the same release                  | `nil`                                          |
-| `db.host`            | database host to connect to               | `nil`          |
-| `ingress.enabled`            | ingress resource to be added              | `false`          |
-| `ingress.annotations`            | ingress annotations              | `{ingress.kubernetes.io/rewrite-target: /,    nginx.ingress.kubernetes.io/rewrite-target: /}`          |
-| `ingress.path`            | path to access frontend               | `/`          |
-| `ingress.host`            | ingress host               | `nil`          |
-| `ingress.tls`            | tls for ingress               | `[]`          |
-| `resources`                          | CPU/Memory resource requests/limits      | `{}`      |
-| `nodeSelector`                   | Node labels for pod assignment             | `{}`                                                    |
-| `tolerations`                    | List of node taints to tolerate            | `[]`                                                    |
-| `affinity`                       | Map of node/pod affinities                 | `{}`                                                    |
+|         Parameter          |               Description                |                         Default                         |
+|----------------------------|------------------------------------------|---------------------------------------------------------|
+| `global.imageRegistry`     | Global Docker image registry             | `nil`                                                   |
+| `image.registry`           | phpMyAdmin image registry                | `docker.io`                                             |
+| `image.repository`         | phpMyAdmin Image name                    | `bitnami/phpmyadmin`                                    |
```

Comment:
This is weird... You used `image` (no capital letters) previously and now you use `Image`. Please be consistent